app/assets/stylesheets/application.scss

```diff
@@ -17,5 +17,48 @@
 @import "bootstrap";
 
 body {
-  background-color: white;
+  background-color: #e5e8e0;
+}
+
+.container-fluid { max-width: 80%;
+
+}
+
+img {width: 100%;
+
+}
+
+.featured {
+  background-color: #c3a17d;
+  h3{color: white;
+    text-transform: uppercase;
+  }
+  h6 {color: #fff;
+    border-bottom: 1px solid #fff;
+  }
+
 }
+
+h1, h2, h3 {font-family: 'Oswald', sans-serif;
+
+}
+
+h6 {
+  font-family: 'Merriweather', serif;
+  font-style: italic;
+}
+
+.featured-two {
+  background-color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.